files/src/index.ts

```diff
@@ -1,5 +1,5 @@
-import App from './main';
 import { ComponentManager, setPropertyDidChange } from '@glimmer/component';
+import App from './main';
```

Comment:
Which rule is forcing this change? I don’t fully understand why the various import statements are moved around...

Reply:
This change is caused by [`ordered-imports`](https://palantir.github.io/tslint/rules/ordered-imports/) which is included in the recommended preset.

Reply:
@rwjblue do you prefer to disable the rule or keep the changes? I'd go with the changes.

Reply:
I’m not a giant fan of this specific rule, but meh.

Reply:
Sadly the recommended preset comes with too much opinions about stylistic choices in my opinion. I prefer what ESLint is doing in their recommended preset (not including any stylistic choices at all) but as far as I know, no similar preset exists for TSLint.